include/ord/Design.h

```diff
@@ -175,6 +189,32 @@ class Design
   bool isSequential(odb::dbMaster* master);
   bool isBuffer(odb::dbMaster* master);
   bool isInverter(odb::dbMaster* master);
+  bool isInPower(odb::dbITerm* iterm);
+  bool isInGround(odb::dbITerm* iterm);
+  sta::ClockSeq findClocksMatching(const char *pattern, bool regexp, bool nocase);
+  sta::Clock* defaultArrivalClock();
+  std::string getITermName (odb::dbITerm* ITerm);
```

Comment:
Design is starting to become the timing API which I think isn't a good idea.  That will be two large APIs lumped into one. Would you split out a timing API and move the timing relevant methods?